src/App.js

```diff
@@ -0,0 +1,29 @@
+import { Provider } from 'react-redux';
+import { Routes, Route, Navigate } from 'react-router-dom';
+import store from './redux/store';
+import './App.css';
+import Sidebar from './components/Sidebar/index';
+import AirPollution from './pages/AirPollution';
+import Contact from './pages/Contact';
+import Dashboard from './pages/Dashboard';
+import Detail from './pages/Detail';
+
+function App() {
```

Comment:
- [ ] As the last code reviewer said 👇🏽 your app must follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), please 🙏🏽 fix it to complete all the project requirements.

> The app should follow the design guidelines including the general layout, kindly make sure to implement this so your app looks like the provided design

> You should follow these [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), including:
> Colors (select one main color for your website).
> Typography: font face, size and weight.
> Layout: composition and space between elements.

Reply:
@franclobo it is not possible to follow the design because my API is based on search and not on filtering the results. Kindly see the comments below. thank you